app/src/main/java/io/neurolab/main/NeuroLab.java

```diff
@@ -175,6 +188,52 @@ protected void onCreate(Bundle savedInstanceState) {
             meditationCard.setVisibility(View.VISIBLE);
             meditationCard.setOnClickListener(this);
         }
+
+        checkForUpdates(appUpdateManager, appUpdateInfoTask);
+    }
+
+    private void checkForUpdates(AppUpdateManager appUpdateManager, Task<AppUpdateInfo> appUpdateInfoTask) {
+        appUpdateInfoTask.addOnSuccessListener(appUpdateInfo -> {
+            if (appUpdateInfo.updateAvailability() == UpdateAvailability.UPDATE_AVAILABLE && appUpdateInfo.isUpdateTypeAllowed(IMMEDIATE)) {
+                try {
+                    appUpdateManager.startUpdateFlowForResult(appUpdateInfo, IMMEDIATE, this, MY_REQUEST_CODE);
+                } catch (IntentSender.SendIntentException e) {
+                    e.printStackTrace();
+                }
+            }
+        });
+    }
+
+    @Override
+    public void onActivityResult(int requestCode, int resultCode, Intent data) {
+        if (requestCode == MY_REQUEST_CODE) {
```

Comment:
Combine both the conditions. There is no else, hence no need to create separate conditions

Reply:
@adityastic Sure! On it!

Reply:
@adityastic  Done!